### PR TITLE
feat: add error logging support through group bot

### PR DESCRIPTION
**Pull request by LukeTheWalker**
Seeing the need to log malformed http request which may cause some silent bugs (e.g. #5) I implemented a rough error handling mechanism hopefully to be integrated with the dmi bot exception forwarder in the telegram group

### CODE REVIEW

File: albo.py [hunk 1]
```diff
@@ -12,23 +12,30 @@ def escape_char (text: str, char_to_escape: List[str] = ['_', '*', '[', '`']) ->
...
   return text
 
+def send_https_request (URL: str) -> None:
+  r = requests.get(url = URL)
+  responseJSON = r.json()
+  if responseJSON["ok"] == False:
+    raise ValueError (json.dumps(responseJSON, indent=2), URL.split("&")[-1])
+
+
 def send_telegram_message(text: str) -> None:
```
Comment (Helias) [inline albo.py:22]:
```suggestion
def send_telegram_message(text: str, chat_id = CHATID) -> None:
```

File: albo.py [hunk 2]
```diff
@@ -12,23 +12,30 @@ def escape_char (text: str, char_to_escape: List[str] = ['_', '*', '[', '`']) ->
...
     params = {
         'chat_id': CHATID,
```
Comment (Helias) [inline albo.py:24]:
```suggestion
        'chat_id': chat_id,
```

File: albo.py [hunk 3]
```diff
@@ -73,9 +80,13 @@ def send_telegram_attachments(pdf_links: List[str]) -> None:
...
-  send_telegram_message(message)
-  attachments = ["http://albo.unict.it/" + list_item['href'] for list_item in tr.find_all('a')]
-  send_telegram_attachments(attachments)
+  try:
+    send_telegram_message(message)
+    attachments = ["http://albo.unict.it/" + list_item['href'] for list_item in tr.find_all('a')]
+    send_telegram_attachments(attachments)
+  except ValueError as err:
+    error_string = "response sent: " + err.args[0] + "\n" + "last parameter: "+ err.args[1]
+    print (error_string) #Insert here bot sending error message in group
```
Comment (Helias) [inline albo.py:89]: you can create a CHAT_ID_DEV and use this
```suggestion
    send_telegram_message(error_string, CHAT_ID_DEV)
```

File: albo.py (hunk 1 above)
Comment (LukeTheWalker) [reply albo.py]: Might as well update all the other methods to have `chat_id` as an argument, for consistency

**Review by Helias — APPROVED** [review]

**Pull request merged**